model/flow/chunk.go

```diff
@@ -367,6 +366,19 @@ type BlockExecutionDataRoot struct {
 	ChunkExecutionDataIDs []cid.Cid
 }
 
+// EncodeRLP defines RLP encoding behaviour for BlockExecutionDataRoot.
```

Comment:
```suggestion
// EncodeRLP defines an RLP encoding BlockExecutionDataRoot. We need to define a custom RLP encoding since [cid.Cid] doesn't have one. Without it we can't produce a collision-resistant hash.
// No errors are expected during normal operations.
```